lang/en/block_html_placeholders.php

```diff
@@ -41,3 +41,9 @@
 profession | Doctor
 type | On-line
 </pre>';
+$string['shortcodes'] = 'Displays the placeholder.';
```

Comment:
Can't see where this is being called, but generally you don't need to add a full-stop to the end of titles/labels, only for descriptions.

Reply:
Well.. all other string do have a full-stop.

![image](https://user-images.githubusercontent.com/4626431/135778808-4824ace6-9b8f-4172-b2aa-01984655b5af.png)

Reply:
Ah okay. Was thinking the string was used in a moodle mod_form but couldn't see how it was linked. All good then.